mods/mc_worldmanager/commands.lua

```diff
@@ -29,13 +23,23 @@ minetest.register_chatcommand("localPos", {
 
 commands["new"] = {
     func = function(name, params)
-        local realmName = params[2]
+        local realmName = tostring(params[1])
         if (realmName == "" or realmName == nil) then
             realmName = "Unnamed Realm"
         end
-        local size = params[3]
-        local sizeY = params[4]
-        local newRealm = Realm:New(realmName, { x = size, y = sizeY, z = size })
+        local sizeX = tonumber(params[2])
```

Comment:
`tonumber(nil)` produces `nil`, so the issue on lines [26-27](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/109/files?diff=unified&w=1#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R26-R27) does not apply here